flexget/components/notify/notifiers/mqtt.py

```diff
@@ -0,0 +1,191 @@
+import json
+
+from loguru import logger
+
+from hashlib import sha256
+import ssl
+
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import DependencyError, PluginWarning
+
+plugin_name = 'mqtt'
+logger = logger.bind(name=plugin_name)
+
+class MQTTNotifier:
+    """
+    Example::
+      notify:
+        entries:
+          via:
+            - mqtt:
+                broker_address: "iot.eclipse.org"
+                topic: "flexget/notifications"
+                [broker_port: 1883]
+                [broker_timeout: 30]
+                [broker_transport: ['tcp','websockets'] ]
+                [broker_protocol: ['MQTTv31', 'MQTTv311'] ]
+                [username: yourUsernameHere]
+                [password: yourPasswordHere]
+                [enable_encrypted_communication: True/False]
+                [certificates:
+                    broker_ca_cert: /path/to/pem/encoded/broker_ca_certificate.crt
+                    client_cert: /path/to/pem/encoded/client_certificate.crt
+                    client_key: /path/to/pem/encoded/client_certificate.key
+                    validate_broker_cert: True/False
+                    tls_version: ['tlsv1.2', 'tlsv1.1', 'tlsv1']
+                ]
+                [qos: [0,1,2] ]
+                [retain: True/False]
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'topic': {'type': 'string'},
+            'broker_address': {'type': 'string'},
+            'broker_port': {'type': 'integer', 'default': 1883},
+            'broker_timeout': {'type': 'integer', 'default': 30},
+            'broker_transport': {'type': 'string', 'default': 'tcp', 'enum': ['tcp', 'websockets']},
+            'broker_protocol': {'type': 'string', 'default': 'MQTTv311', 'enum': ['MQTTv31', 'MQTTv311']},
+            'username': {'type': 'string'},
+            'password': {'type': 'string'},
+            'enable_encrypted_communication': {'type': 'boolean', 'default': False},
```

Comment:
Drop the enable prefix, not the convention

Reply:
Done in commit 793d9e5 to format plugin with Black